uavcan/internet/2500.OutgoingUDPPacket.uavcan

```diff
@@ -0,0 +1,121 @@
+# modem node be exhausted, it is allowed to remove old NAT entries earlier, following the policy of
+# least recent use.
+#
+# Having received a UDP packet from a remote host, the modem node would check the NAT table in order
+# to determine where on the UAVCAN bus the received data should be forwarded to. If the NAT table
+# contains no matches, the received data should be silently dropped. If a match is found, the
+# modem node will forward the data to the recipient node using the service HandleIncomingUDPPacket.
+# Whether the invocation of the service is successful or not is of no concern to the modem node.
+# Please refer to the definition of the service data type for a mode in depth review of the reverse
+# forwarding process.
+#
+# Modem nodes can also perform traffic shaping, shall it be needed, by means of delaying or
+# dropping UDP forwarding requests that exceed the quota.
+#
+# To summarize, a typical data exchange occurrence should amount to the following actions:
+#
+#   - A local UAVCAN node broadcasts a message of type OutgoingUDPPacket with the payload it needs
+#     to forward. If the node expects the remote host to send any data back, it sets the
+#     masquerading flag.
+#
+#   - Every modem node on the bus receives the message and performs the following actions:
+#
+#       - The domain name is resolved, unless the destination address provided in the message
+#         is already an IP address, in which case this step should be skipped.
+#
+#       - The domain name to IP address matching is added to a local DNS cache, although this
+#         part is entirely implementation defined.
+#
+#       - The masquerading flag is checked. If it is set, a new entry is added to the NAT table.
+#         If such entry already existed, its expiration timeout is reset. If no such entry existed
+#         and a new one cannot be added because of memory limitations, the least recently used
+#         (i.e. oldest) entry of the NAT table is replaced with the new one.
+#
+#       - The payload is forwarded to the determined IP address.
+#
+#   - At this point, forwarding is complete. Should any of the modem nodes receive an incoming
+#     packet, they would attempt to perform reverse forwarding according to the above provided
+#     algorithm.
+#
+
+#
+# Modem nodes are required to keep NAT table entries for at least this amount of time,
+# unless the table is overflowed, in which case it is allowed to remove least recently used
+# entries in favor of newer ones.
+# Modem nodes are required to be able to accommodate at least 100 entries in the NAT table.
+# The units are seconds.
+#
+uint32 NAT_ENTRY_TTL = 86400
+
+#
+# This field is set to an arbitrary value by the transmitting node in order to be able to
+# match the response with the locally kept context. The function of this field is virtually
+# identical to that of UDP/IP port numbers. Modem nodes do not care about specific values
+# kept in this field.
```

Comment:
I think they should care and if a new session ID is given with NAT enabled, then a new source port should be used.